src/dunes/__main__.py

```diff
@@ -6,11 +6,11 @@
 from args import parse_optional
 import version_selector
 from docker import docker_error
-from dune import dune
-from dune import dune_error
-from dune import dune_node_not_found
-from dune import node
-from dune import version_full
+from dunes import dunes
+from dunes import dunes_error
+from dunes import dunes_node_not_found
+from dunes import node
+from dunes import version_full
 
 
 def handle_version():
```

Comment:
handle version needs an update to:
`print("DUNES " + version_full())`

Reply:
Thanks for your help. Renamed.